read-the-docs/source/JIRA.rst

```diff
@@ -123,13 +123,23 @@ Adds a comment to a JIRA ticket.
 change_status()
 ---------------
 
-``change_status(self, status)``
+``change_status(self, status, **kwargs)``
 
 Changes status of a JIRA ticket.
+With the future state of JIRA, no project will automatically set resolution when changing the status of the ticket to RESOLVED.
+We need to pass additional fields like 'resolution' or 'comment' to resolve the tickets via automation using TicketUtil itself.
```

Comment:
This statement is true for our specific Jira instance but is not necessarily true for other Jira instances out there. What about changing this to:

`Some Jira instances may require a resolution or comment when transitioning tickets to a resolved state. See examples below.`

And then let's also include the original example to 'In Progress' to show that some states don't need extra kwargs. This example can be put above the new description you wrote, right after "Changes status of a Jira ticket.":

`t = ticket.change_status('In Progress')`

Reply:
The changes have been incorporated in the new commit